README.md

```diff
@@ -1,2 +1,66 @@
 # rokt-demo-ios
-Rokt iOS Demo Application
+Rokt Demo application is a sample app built to showcase Rokt SDK functionality.
+
+## Requirements
+
+XCode 13 and above. Project is configured to run on iOS 13.0 and above.
+
+## Project architecture
+
+This project it is implemented based on MVVM pattern with SwiftUI.
+It contains UI, Model, Services, Utils and Resources.
+
+View and ViewModels are located in the UI Group.
+Services represents the network logics of the application and model represnts the data.
+Utils are shared between Groups of the application
+All the other resources of the application is located in Resources. 
+![UI diagram](diagram.png)
+
+## Where to start
+
+As it is defined on SceneDelegate, the starting point of the application is HomePageView located in `UI/Home` folder.
+
+## How to release the app
+- Update `Version` and `Build` in RoktDemo target
+- Commit & push and merge it to the release branch
+- Aprove the **hold** on **CircleCI** and wait till it finished
```

Comment:
link to circleCI

Reply:
Thanks for mentioning, I would put the link to CircleCI but not the actual build